packages/shared-ux/link/redirect_app/impl/src/redirect_app_links.container.tsx

```diff
@@ -23,7 +24,7 @@ import { RedirectAppLinks as Component } from './redirect_app_links.component';
  * ```
  */
 export const RedirectAppLinks: FC<{ className?: string }> = ({ className, children }) => (
-  <Component {...useServices()} className={className}>
+  <Component css={redirectAppLinksStyles} {...useServices()} className={className}>
```

Comment:
I took a closer look a this, are these styles intended to applied on the `div` wrapping the inner components?

Reply:
Looks like I am able to reproduce the vertical issue when zoomed out in the browser

<img width="731" alt="Screenshot 2022-11-08 at 13 12 41" src="https://user-images.githubusercontent.com/8155004/200560846-c88bb864-da8c-40c0-b5f2-50b430216bb1.png">

Reply:
The issue is gone for me now, running Kibana build on commit 99f1d0799922f25068289984fc69109b27c034df (main from a few hours ago).

The flamegraph now fills the vertical space in FF and Chrome.